distribution/src/test/java/com/hazelcast/distribution/DistributionIT.java

```diff
@@ -0,0 +1,125 @@
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category(QuickTest.class)
+public class DistributionIT {
+    private static final String BASE_NAME = "hazelcast-" + getHzVersion();
+    private static final String DISTRIBUTION_FILE = "target/" + BASE_NAME + ".tar.gz";
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    private Process hzProcess;
+    private String hazelcastHome;
+
+    @Test
+    public void should_start_instance_with_hz_command() throws Exception {
+        assumeThatNoWindowsOS();
+        String tmpDirectory = temporaryFolder.newFolder().getAbsolutePath();
+        untarDistribution(tmpDirectory);
+        hazelcastHome = tmpDirectory + "/" + BASE_NAME;
+
+        String[] startInstanceCmd = {hazelcastHome + "/bin/hz", "start", "-c", "src/test/resources/integration-test-hazelcast.yaml"};
+        hzProcess = executeAsync(startInstanceCmd);
+
+        await().atMost(30, TimeUnit.SECONDS)
+                .ignoreExceptions()
+                .until(DistributionIT::memberIsHealthy);
+    }
+
+    private static boolean memberIsHealthy() {
+        int port = getFirstMemberPort();
+        return given().baseUri("http://127.0.0.1:" + port + "/hazelcast/health/ready")
+                .when().get()
+                .then().extract().statusCode() == SC_OK;
+    }
+
+    private static int getFirstMemberPort() {
+        ClientConfig clientConfig = new ClientConfig();
+        clientConfig.setClusterName("hz-it-cluster");
+        clientConfig.getNetworkConfig().addAddress("127.0.0.1");
+        HazelcastInstance client = HazelcastClient.newHazelcastClient(clientConfig);
+        try {
+            Member firstMember = client.getCluster().getMembers().iterator().next();
+            return firstMember.getAddress().getPort();
+        } finally {
+            client.shutdown();
+        }
+    }
+
+    private static String getHzVersion() {
+        Properties properties = new Properties();
+        try (FileInputStream fileInputStream = new FileInputStream("target/maven-archiver/pom.properties")) {
+            properties.load(fileInputStream);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return properties.getProperty("version");
+    }
+
+    private static void untarDistribution(String targetDirectory) throws InterruptedException, IOException {
+        int unpackingExitCode = execute("tar", "-xzvf", DISTRIBUTION_FILE, "-C", targetDirectory);
+        assertThat(unpackingExitCode).describedAs("untar should be successful").isZero();
+    }
+
+    private static int execute(String... commandAndArgs) throws InterruptedException, IOException {
+        return executeAsync(commandAndArgs).waitFor();
+    }
+
+    private static Process executeAsync(String... commandAndArgs) throws IOException {
+        ProcessBuilder processBuilder = new ProcessBuilder(commandAndArgs);
+        Map<String, String> environment = processBuilder.environment();
```

Comment:
The test should disable phone homes.

```suggestion
        Map<String, String> environment = processBuilder.environment();
        environment.put("JAVA_OPTS", "-Dhazelcast.phone.home.enabled=false");
```